usaspending_api/references/models.py

```diff
@@ -400,6 +408,43 @@ class Meta:
         unique_together = (('recipient_unique_id'),)
 
 
+class LegalEntityOfficers(models.Model):
+    legal_entity = models.OneToOneField(
+        LegalEntity, on_delete=models.CASCADE,
+        primary_key=True, related_name='officers')
+
+    officer_1_name = models.TextField(null=True, blank=True)
+    officer_1_amount = models.DecimalField(max_digits=20, decimal_places=2, blank=True, null=True)
+    officer_2_name = models.TextField(null=True, blank=True)
+    officer_2_amount = models.DecimalField(max_digits=20, decimal_places=2, blank=True, null=True)
+    officer_3_name = models.TextField(null=True, blank=True)
+    officer_3_amount = models.DecimalField(max_digits=20, decimal_places=2, blank=True, null=True)
+    officer_4_name = models.TextField(null=True, blank=True)
+    officer_4_amount = models.DecimalField(max_digits=20, decimal_places=2, blank=True, null=True)
+    officer_5_name = models.TextField(null=True, blank=True)
+    officer_5_amount = models.DecimalField(max_digits=20, decimal_places=2, blank=True, null=True)
+
```

Comment:
In all future cases, the Officer object is created at the same time as the LegalEntity object; but we need update_date for the ETL process